__tests__/app.test.js

```diff
@@ -0,0 +1,22 @@
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection.js");
+const testData = require("../db/data/test-data/index.js");
+const seed = require("../db/seeds/seed");
+
+beforeEach(() => seed(testData));
+afterAll(() => db.end());
+
+describe("/api/topics", () => {
+  test("GET:200 returns an array of topic objects with slug and description properties", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((response) => {
+        response.body.topics.forEach((topic) => {
+          expect(typeof topic.description).toBe("string");
+          expect(typeof topic.slug).toBe("string");
+        });
+      });
+  });
+});
```

Comment:
Good test description and well setup, You may consider adding an array length check assertion here also. Not compulsory though.